src/components/slider/context/SliderGroup.js

```diff
@@ -21,22 +21,9 @@ export default class SliderGroup extends Component {
     };
   }
 
-  // static getDerivedStateFromProps(nextProps, prevState) {
-  //   if (nextProps.color !== Colors.getHexString(prevState.value)) {
-  //     return {value: Colors.getHSL(nextProps.color)};
-  //   }
-  //   return null;
-  // }
-
-  UNSAFE_componentWillReceiveProps(nextProps) {
-    if (nextProps.color !== Colors.getHexString(this.state.value)) {
-      return {value: Colors.getHSL(nextProps.color)};
-    }
-  }
-
```

Comment:
Doesn't it break functionality?

Reply:
Not when I checked it. Did you have an issue?

Reply:
No. 
But If I understand correctly.. 
It supported initializing the color value when passing a different value to the prop. No? 

Actually, I think it's better when removed....
merging.